persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxGenerator.java

```diff
@@ -1054,4 +1058,11 @@ private static String getEntityNameConstant(String entityName) {
         }
         return outputString.toString();
     }
+
+    private static String formatFieldName(String fieldName) {
```

Comment:
```suggestion
    private static String stripEscapeCharacter(String fieldName) {
```

Reply:
updated in https://github.com/ballerina-platform/persist-tools/pull/117/commits/06860b7f7fa6a73ba0dfad04d67d653c21f63dd1